src/ClientRuntime/Microsoft.Rest.ClientRuntime.Tests/JsonTransformationConverterTest.cs

```diff
@@ -353,6 +354,28 @@ public void TestDeserializationOfResourceWithConflictingProperties()
 }";
             var newJson = JsonConvert.SerializeObject(deserializedResource, deserializeSettings);
             Assert.Equal(expectedSerializedJson, newJson);
-        }        
+        }
+
+        [Fact]
+        public void TestPolymorphicJsonDeserializer()
```

Comment:
what about testing serialization? I assume there is an equivalently significant impact on that and you've touched `PolymorphicSerializeJsonConverter.cs` which I guess was covered by this test! ;-)